# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/DecodedSurfaceProvider.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c image/DecodedSurfaceProvider.cpp
--- a/image/DecodedSurfaceProvider.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/image/DecodedSurfaceProvider.cpp	Fri May 24 11:32:54 2019 +0000
@@ -5,7 +5,7 @@
 
 #include "DecodedSurfaceProvider.h"
 
-#include "gfxPrefs.h"
+#include "mozilla/StaticPrefs.h"
 #include "nsProxyRelease.h"
 
 #include "Decoder.h"
@@ -200,7 +200,7 @@
 }
 
 bool DecodedSurfaceProvider::ShouldPreferSyncRun() const {
-  return mDecoder->ShouldSyncDecode(gfxPrefs::ImageMemDecodeBytesAtATime());
+  return mDecoder->ShouldSyncDecode(StaticPrefs::ImageMemDecodeBytesAtATime());
 }
 
 }  // namespace image